icu4c/source/test/testdata/structLocale.txt

```diff
@@ -11,6 +11,7 @@
 // Since ICU doesn't follow the traditional fallback model where all keys fallback to root,
 // root can't be used to validate the structure.
 // This file was generated from en.txt, and all ".*" were replaced with "".
```

Comment:
As per FIXME below, I'm not trusting this line is up to date? How does one "generate" this file?

Reply:
Responding to the above and to
> @pedberg-icu since you seem to be the primary maintainer of structLocale.txt - how does this file work? Is it manually maintained?

Yes, structLocale.txt is manually maintained these days. Basically it needs an empty entry corresponding to any entry that might occur in any locale. So for example if some locale has a gregorian-calendar availableFormats entry for skeleton MMMMEEEEd, structLocale.txt needs such an entry (with empty string for value), even though en does not have such an entry.

Reply:
I have changed instructions at the top of the file: I removed this line and added mention of "manually maintained".